hm...seems most tests are now actually failing with
```
 java.lang.BootstrapMethodError: java.lang.NoSuchMethodError: org.apache.commons.io.IOUtils.byteArray()[B
Caused by: java.lang.NoSuchMethodError: org.apache.commons.io.IOUtils.byteArray()[B
```
likely because of some dependency update, maybe here https://github.com/uc-cdis/WebAPI/commit/06067d50296044f7373aee6ca1bf3e3430436c2b

--- a/.github/workflows/ci_custom.yaml
+++ b/.github/workflows/ci_custom.yaml
@@ -1,0 +1,38 @@
+# Continuous integration
+name: CI CTDS
+
+# Run customized version of CI in our local fork's main (default) branch and pull requests to
+# this branch (note: this ci_custom.yaml file is a simplified version of the ci.yaml file in this same directory)
+on:
+  push:
+    branches: [ 2.15.0-DEV ]
+  pull_request:
+    branches: [ 2.15.0-DEV ]
+
+jobs:
+  # Build and test the code
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      MAVEN_PROFILE: webapi-postgresql
+
+    steps:
+      # Checks-out repository under $GITHUB_WORKSPACE, so the job can access it
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v1
+        with:
+            java-version: 8
+
+      - name: Maven cache
+        uses: actions/cache@v2
+        with:
+            path: ~/.m2
+            # Key for restoring and saving the cache
+            key: ${{ runner.os }}-maven-${{ hashFiles('pom.xml') }}
+            restore-keys: |
+              ${{ runner.os }}-maven-
+
+      - name: Test
+        run: mvn -B -P${{ env.MAVEN_PROFILE }} test

--- a/pom.xml
+++ b/pom.xml
@@ -831,7 +831,7 @@
     <dependency>
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
-      <version>2.7</version>
+      <version>2.14.0</version>
     </dependency>
     <dependency>
       <groupId>com.sun.xml.security</groupId>

--- a/src/test/java/org/ohdsi/webapi/generationcache/GenerationCacheTest.java
+++ b/src/test/java/org/ohdsi/webapi/generationcache/GenerationCacheTest.java
@@ -4,6 +4,7 @@ import com.odysseusinc.arachne.commons.types.DBMSType;
 import com.odysseusinc.arachne.execution_engine_common.api.v1.dto.KerberosAuthMechanism;
 import net.lingala.zip4j.ZipFile;
 import net.lingala.zip4j.exception.ZipException;
+import org.springframework.transaction.annotation.Transactional;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
@@ -170,6 +171,7 @@ public class GenerationCacheTest extends AbstractDatabaseTest {
     }
 
     @Test
+    @Transactional
     public void checkCachingWithEmptyResultSet() {
 
         CacheableGenerationType type = CacheableGenerationType.COHORT;

